content/millennium/r4/base/entities/device.md

```diff
@@ -56,35 +56,25 @@ Search for Devices that meet supplied query parameters:
 --------------|-------------------|---------------|------------------------------------------------------------------------
  `_id`        | This or `patient` | [`token`]     | The logical resource id associated with the Device. Example: `7890`
  `patient`    | This or `_id`     | [`reference`] | The patient to whom the device is affixed. Example: `12345`
-`_revinclude` | N                 | [`token`]     | Provenance resource entries to be returned as part of the bundle. Example: `_revinclude=Provenance:target`
+`_revinclude` | No                | [`token`]     | Provenance resource entries to be returned as part of the bundle. Example: `_revinclude=Provenance:target`
```

Comment:
I can't remember, does this match the other ones we've done so far?

Reply:
I think I've been doing "N"/"Y" on most.

For what it's worth, I expected that these sections would need to get ripped apart when we get new formatting guidelines, so didn't have it as a high priority.

Reply:
Reverted it back to N